cmd/orb-server/startcmd/start.go

```diff
@@ -145,16 +140,37 @@ func startOrbServices(parameters *orbParameters, srv server) error {
 	casClient := cas.New(parameters.casURL)
 	opStore := mocks.NewMockOperationStore()
 
+	// TODO: For now fetch signing public key from local KMS (this will handled differently later on: webfinger or did:web)
```

Comment:
So this temporarily means the configured KMS needs to knows all the public keys?

Reply:
Yes, this is temporary. I made a comment that we will have to have different fetcher solution.